Modules/Office365DSC/DSCResources/MSFT_SCComplianceCase/MSFT_SCComplianceCase.schema.mof

```diff
@@ -4,6 +4,6 @@ class MSFT_SCComplianceCase : OMI_BaseResource
     [Key, Description("The Name parameter specifies the unique name of the compliance case.")] String Name;
     [Write, Description("The description of the case.")] String Description;
     [Write, Description("Specify if this case should exist or not."), ValueMap{"Present","Absent"}, Values{"Present","Absent"}] String Ensure;
-    [Write, Description("Status for the case. Can either be 'Active' or 'Closed'"), ValueMap{"Active","Closed"}, Values{"Active","Closed"}] String Status;
+    [Write, Description("Status for the case. Can either be 'Active' or 'Closed'"), ValueMap{"Active","Closed", "ClosedWithError"}, Values{"Active","Closed", "ClosedWithError"}] String Status;
```

Comment:
Do we need to update the Description to also have 'ClosedWithError' status?

Reply:
update the description.